### PR TITLE
Sponsors: Add Flatcar sponsors to readme.

### PR DESCRIPTION
**Pull request by t-lo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,3 +60,27 @@ If you would like to start contributing to an issue or PR, please assign it to y
 ### Major Contributions
 
 We will publish a roadmap discussing long-term goals of the project very soon - stay tuned!
+
+# Sponsors
+
+## Development Sponsorr
```
Comment (invidian) [inline README.md]:
```suggestion
## Development Sponsors
```

**Review by invidian — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (t-lo) [reply README.md]: Nice catch, thanks!

**Comment by t-lo** [comment]
Addressed @invidian 's feedback.
Please note that the Github Diff document preview breaks image URLs; please use my PR branch directly for previewing the resulting README: https://github.com/flatcar-linux/Flatcar/tree/t-lo/add-sponsors

**Review by invidian — APPROVED** [review]
OK

File: README.md [hunk 2]
```diff
@@ -58,3 +60,27 @@ If you would like to start contributing to an issue or PR, please assign it to y
...
+
+
+<img src="sponsors/kinvolk_400x400.jpg" width="100pt" alt="Kinvolk.io logo" title="Kinvolk.io logo" />
+
+[Kinvolk.io](https://www.kinvolk.io/)
+
+
+<img src="sponsors/gs-logo-on-dark.svg" width="450pt" alt="Giant Swarm logo" title="Giant Swarm logo">
+
+[Giant Swarm](https://www.giantswarm.io/)
```
Comment (joaquimrocha) [inline README.md]: Maybe adding the sponsors in a table is better, as it allows the list to scale better without expanding the README vertically too much.
Also, should we just have links directly from the images? Sounds a bit redundant to have the labels with the names when the logos and titles already have that.

**Review by joaquimrocha — COMMENTED** [review]
Left a comment about how the sponsors are organized.
Not setting as requesting changes because it's not something that's wrong in the pure definition of it nor my decision.

**Pull request merged**